papers/vyas_ramasubramani/94_vyas_ramasubramani.rst

```diff
@@ -188,6 +191,8 @@ Additionally, at any point we can get an overview of what the implicit data spac
      'v': 'int([1, 2, 3], 3)',
     }
 
+Additionally, schema can be filtered, nested keys can be compressed to specified depths, and the number of entries shown in each range can be limited as desired.
```

Comment:
You should mention that all keys that have *constant* values across all data points can be automatically filtered so as to only see "relevant" schema data.